File: PHP/fizzbuzz.php
<?php

for ($i = 0; $i <= 1000000; ++$i) {
    if (0 === $i % 15) {
        echo 'FizzBuzz'.PHP_EOL;
    } elseif (0 === $i % 3) {
        echo 'Fizz'.PHP_EOL;
    } elseif (0 === $i % 5) {
        echo 'Buzz'.PHP_EOL;
    } else {
        echo $i.PHP_EOL;
    }
}

File: Go/println/fizzbuzz.go
package main

import "fmt"

func main() {
	for i := 1; i <= 1000000; i++ {
		if i % 15 == 0 {
			fmt.Println(i, ":", "FizzBuzz")
		} else if i % 3 == 0 {
			fmt.Println(i, ":", "Fizz")
		} else if i % 5 == 0 {
			fmt.Println(i, ":", "Buzz")
		} else {
			fmt.Println(i, ":", i)
		}
	}
}